scripts/i18n/sync.js

```diff
@@ -126,7 +181,33 @@ async function syncTranslationRepo(code) {
   shell.exec(`git remote add source ${sourceRepoGitUrl}`)
   shell.exec(`git fetch source master`)
 
-  // TODO don't run the sync script if there is a current PR from the bot
+  const repository = await getRepository(owner, transRepoName)
+
+  if (repository.syncPullRequests.nodes.length > 0) {
+    logger.info(
+      `There are currently open sync pull requests. Please ask the language maintainers to merge the existing PR(s) in before opening another one. Exiting...`
+    )
+    process.exit(0)
+  }
+
+  logger.info(`No currently open sync pull requests.`)
+
+  let syncLabel
+  if (!repository.syncLabel) {
+    logger.info(
+      `Repository does not have a "${syncLabelName}" label. Creating one...`
+    )
+    syncLabel = await createLabel({
+      repositoryId: repository.id,
+      name: syncLabelName,
+      description: `Sync with translation source. Used by @gatsbybot to track open sync pull requests.`,
+      color: `fbca04`,
+    })
+  } else {
+    logger.info(`Repository has an existing ${syncLabelName} label.`)
+    syncLabel = repository.syncLabel
+  }
+  process.exit(0)
```

Comment:
Since I'm not sure whether this is finished or not ... was this exit intentional? Temporary? The rest of this function is dead code now.

Reply:
whoops, fixed